amundsen_application/static/js/components/Tags/TagsList/styles.scss

```diff
@@ -0,0 +1,35 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+
+hr.header-hr {
```

Comment:
could you remove the 'hr'? It is usually a good practice to only style on classes, it makes specificity much more simple.